app/components/school/school_details_summary_list_component.rb

```diff
@@ -0,0 +1,84 @@
+class School::SchoolDetailsSummaryListComponent < ViewComponent::Base
+  validates :school, presence: true
+
+  delegate :school_will_order_devices?,
+           :school_contact,
+           to: :preorder_information
+
+  def initialize(school:)
+    @school = school
+  end
+
+  def rows
+    [
+      {
+        key: 'Status',
+        value: render(SchoolPreorderStatusTagComponent.new(school: @school)),
+      },
+      {
+        key: 'Provisional allocation',
+        value: pluralize(@school.std_device_allocation&.allocation.to_i, 'device'),
+      },
+      {
+        key: 'Type of school',
+        value: @school.type_label,
+      },
+      {
+        key: 'Who will order?',
+        value: "The #{(@school.preorder_information || @school.responsible_body).who_will_order_devices_label.downcase} orders devices",
+      },
+    ] + school_contact_row_if_contact_present + chromebook_rows_if_needed
```

Comment:
We don't need:
- school contact row
- who will order row

The only extra rows from the prototype we need are the chromebook ones.

Reply:
done